pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -162,7 +163,23 @@ let
                 postFetch = "substituteInPlace $out --replace __aarch64__ SK_CPU_ARM64";
                 sha256    = "018fbdzyw9rvia8m0qkk5gv8q8gl7x34rrjbn7mi1fgxdsayn22s";
               }
-            );
+           ) ++ optional ungoogled
+             (fetchgit {
+               url    = "https://github.com/Eloston/ungoogled-chromium.git";
+               rev    = "46bbd9f6686dd1ee2fa67ac94ddd618d0ae56d3b";
```

Comment:
Is it possible to use releases from here? https://github.com/Eloston/ungoogled-chromium/releases

Reply:
Yes